modules/nlb/main.tf

```diff
@@ -0,0 +1,42 @@
+resource "aws_lb" "nlb" {
```

Comment:
I can't recall if we discussed it, but do we need `enable_cross_zone_load_balancing` here?

Reply:
done